package.json

```diff
@@ -161,7 +161,9 @@
 		"build": "npm run build:packages && cross-env NODE_ENV=production webpack",
 		"check-engines": "check-node-version --package",
 		"check-licenses": "concurrently \"wp-scripts check-licenses --prod --gpl2\" \"wp-scripts check-licenses --dev\"",
-		"ci": "concurrently \"npm run lint\" \"npm run test-unit:coverage-ci\"",
+		"precheck-local-changes": "npm run docs:build",
+		"check-local-changes": "git diff --exit-code > /dev/null || ( echo \"There are local uncommitted changes after one or both of 'npm install' or 'npm run docs:build'!\" && exit 1 );",
```

Comment:
Yeah, especially in Travis, it's quite difficult to parse the failing output, made worse with `concurrently` which doesn't display the failing command in the final result (https://github.com/kimmobrunfeldt/concurrently/issues/134).

For this case specifically, it would be nice too if we could limit the output by `npm`, such that the message is the only thing or at least more obvious.